Celeste.Mod.mm/Mod/Helpers/ModUpdaterHelper.cs

```diff
@@ -145,10 +188,10 @@ private class MostRecentUpdatedFirst : IComparer<ModUpdateInfo> {
         /// Retrieves the mod updater database location from everestapi.github.io.
         /// This should point to a running instance of https://github.com/max4805/EverestUpdateCheckerServer.
```

Comment:
While you're there, maybe update the github link

Reply:
good point 😅